presto-main/src/test/java/com/facebook/presto/sql/planner/TestQuantifiedComparison.java

```diff
@@ -96,8 +96,7 @@ private void assertQuantifiedComparison(String query)
         assertPlan(query, anyTree(
                 node(JoinNode.class,
                         tableScan("orders"),
-                        anyTree(
```

Comment:
I'm not sure about the complexity of this task. For sure I wouldn't like to do this in this PR